pkg/model/printer_column.go

```diff
@@ -136,10 +137,18 @@ func (r *CRD) addPrintableColumn(
 		name = field.FieldConfig.Print.Name
 	}
 
+	// Fields/Columns with priority 0 are shown in standard view, and those with
+	// priority greater than 0 are shown only in wide view.
+	priority := 1
```

Comment:
Let's use the default of 0, which makes sense... if the user specifies a printcolumn tag, they mean for the column to be shown in standard views.

Reply:
Done by switching to simple `int` type